templates/telecommunications/index.html

```diff
@@ -54,44 +57,32 @@ <h2>Why choose Canonical?</h2>
 </section>
 
 <section class="p-strip--light is-bordered">
-  <div class="row">
-    <div class="col-8">
-      <h2>The complete cloud automation toolset</h2>
-      <p>Operational discipline &mdash; built on automation and repeatability &mdash; has long been crucial to the profitability of CSPs. Canonical applies the same principles to the way clouds are built and run.  From design to deployment and operations, every aspect is managed with Canonical&rsquo;s proven automation toolset, including provisioning tool MAAS, management tool Landscape, and application modeling tool Juju.</p>
-      <ul class="p-list">
+  <div class="row u-equal-height">
+    <div class="col-6 p-card--highlighted">
+      <h2 class="p-card__title">The complete cloud automation toolset</h2>
+      <p class="p-card__content">Operational discipline &mdash; built on automation and repeatability &mdash; has long been crucial to the profitability of CSPs. Canonical applies the same principles to the way clouds are built and run.  From design to deployment and operations, every aspect is managed with Canonical&rsquo;s proven automation toolset, the: provisioning tool MAAS, management tool Landscape, and application modeling tool Juju.</p>
+      <ul class="p-card__content p-list">
         <li><a href="/cloud/juju">Find out more about Juju&nbsp;&rsaquo;</a></li>
         <li><a href="/server/maas">Find out more about MAAS&nbsp;&rsaquo;</a></li>
       </ul>
     </div>
-  </div>
-</section>
-
-<section class="p-strip--light is-bordered">
-  <div class="row">
-    <div class="col-8">
-      <h2>NFVI cloud build</h2>
-      <p>Canonical&rsquo;s Foundation Cloud Build for telcos is a pre-tested, pre-integrated OpenStack cloud built using a proven reference architecture and tailored to the needs of CSPs. Think of it as a fixed price, NFV-ready cloud, deployed by Canonical on your premises in just a few weeks.</p>
-      <p><a href="/cloud/foundation-cloud">Find out more<span class="u-off-screen"> about NFVI cloud build</span>&nbsp;&rsaquo;</a></p>
+    <div class="col-6 p-card--highlighted">
+      <h2 class="p-card__title">NFVI cloud build</h2>
+      <p class="p-card__content">Canonical&rsquo;s Foundation Cloud Build for telcos is a pre-tested, pre-integrated OpenStack cloud built using a proven reference architecture and tailored to the needs of CSPs.</p>
+      <p class="p-card__content">Think of it as a fixed price, NFV-ready cloud, deployed by Canonical on your premises in just a few weeks.</p>
+      <p class="p-card__content"><a href="/cloud/foundation-cloud">Find out more<span class="u-off-screen"> about NFVI cloud build</span>&nbsp;&rsaquo;</a></p>
```

Comment:
I think it would be tidier if these p's were placed inside a div with `p-card__content`